### ISSUE TITLE
Rename and more diverse examples

### ISSUE THREAD

**Issue by jaydenseric** [jaydenseric/apollo-upload-examples#6]
It's time to rethink the name and contents of this repo.

- Apollo should be removed from the name, as uploads are not exclusive to Apollo Server and Client. A rename is also likely for `apollo-upload-server`: https://github.com/jaydenseric/apollo-upload-server/issues/68. It might make sense to rename the project `graphql-mutipart-request-spec-examples` (or something similar but shorter, like `graphql-upload-examples`). 
- Apollo Server now has `apollo-upload-server` built in, so we need to use something else like `graphql-api-koa` or `express-graphql` to demonstrate a manual setup.

A new project structure could include a variety of cross-compatible server and client examples, centering around the [GraphQL multipart request spec](https://github.com/jaydenseric/graphql-multipart-request-spec).

- Server:
  - apollo-upload-server
     - With graphql-api-koa
     - With express-graphql
  - apollo-server
- Client:
  - apollo-upload-client
  - graphql-react

It would be great to have examples for each with common cloud storage providers such as AWS S3.

**Comment by fgcui1204** [comment]
looking forward to that. Thanks very much @jaydenseric

**Comment by dobbbri** [comment]
I make this examples with Vue.js

Vue.js + Apollo + express
https://github.com/dobbbri/vue-apollo-express-upload

Vue.js + Apollo + koa
https://github.com/dobbbri/vue-apollo-server-koa-uplload

Vue.js + Apollo-server
https://github.com/dobbbri/vue-apollo-upload

**Comment by jaydenseric** [comment]
For now we'll keep this repo as just Apollo examples.

It would be a huge maintenance burden to somehow have examples for every framework multiplied by however many storage options; and I would not be able to quality control examples for frameworks and cloud services I don't have experience with.

I'm pretty happy with [the code examples in the `graphql-upload` readme](https://github.com/jaydenseric/graphql-upload#examples), and prefer not to duplicate such effort across multiple repos.

The [GraphQL multipart request spec](https://github.com/jaydenseric/graphql-multipart-request-spec) has become well adopted with [over 20 client/server implementations](https://github.com/jaydenseric/graphql-multipart-request-spec#implementations). It’s now up to the individual frameworks and commercial offerings to advertise and document their support for GraphQL file uploads.

**Comment by thorizer** [comment]
@jaydenseric  any chance I can find a full express-graphql example elsewhere ?

**Comment by jaydenseric** [comment]
@thorizer Not that I'm aware of, although someone probably has one somewhere. The setup is really easy though, and there are code examples in the [`graphql-upload`](https://github.com/jaydenseric/graphql-upload) readme:

https://github.com/jaydenseric/graphql-upload#function-graphqluploadexpress

```js
const express = require('express');
const graphqlHTTP = require('express-graphql');
const { graphqlUploadExpress } = require('graphql-upload');
const schema = require('./schema');

express()
  .use(
    '/graphql',
    graphqlUploadExpress({ maxFileSize: 10000000, maxFiles: 10 }),
    graphqlHTTP({ schema })
  )
  .listen(3000);
```

Use [`GraphQLUpload`](https://github.com/jaydenseric/graphql-upload#class-graphqlupload) in your schema, there are code examples in it's API documentation.

The way the file `Upload` scalar promises are handled in resolvers is exactly the same as the examples here.

**Comment by thorizer** [comment]
@jaydenseric . yes, I've already did the middleware part and the schema. I'm also getting the file metadata returned to me successfully on postman and the client.
however the file is not stored in the filesystem. something wrong with the storeUpload function. 
I did everything the same like the koa example except the "Disable the built in file upload implementation" part since I'm not using the ApolloServer from koa. 

`new ApolloServer({
  // Disable the built in file upload implementation that uses an outdated
  // `graphql-upload` version, see:
  // https://github.com/apollographql/apollo-server/issues/3508#issuecomment-662371289
  uploads: false,
  schema,
  context: { db, storeUpload },
}).applyMiddleware({ app });`

this is my server.js in a [gist](https://gist.github.com/thorizer/932190203f4c06367271956e7f7b3ddf)
